scripts/sct_register_to_template.py

```diff
@@ -364,7 +368,23 @@ def main(args=None):
             # apply straightening
             sct.run(['sct_apply_transfo', '-i', ftmp_seg, '-w', 'warp_curve2straight.nii.gz', '-d', 'straight_ref.nii.gz', '-o', add_suffix(ftmp_seg, '_straight')])
         else:
-            import sct_straighten_spinalcord
+            from sct_straighten_spinalcord import SpinalCordStraightener
+            sc_straight = SpinalCordStraightener(ftmp_seg, ftmp_seg)
+            sc_straight.output_filename = add_suffix(ftmp_seg, '_straight')
+            sc_straight.path_output = './'
+            sc_straight.qc = '0'
+            sc_straight.remove_temp_files = remove_temp_files
+            sc_straight.verbose = verbose
+
+            if vertebral_alignement:
+                sc_straight.centerline_reference_filename = ftmp_template_seg
```

Comment:
ideally we would use the [standardized variable name](https://github.com/neuropoly/spinalcordtoolbox/wiki/variables#variables) for files, path etc.  but that could also be fixed in a subsequent PR

Reply:
This will be done in a future PR.